test/pipelines/test_batch_generation.py

```diff
@@ -122,7 +131,23 @@ def test_get_artman_config_filenames_wildcard():
         'test/cli/data/gapic/core/artman_core.yaml',
     ]
     actual = batch_generation._get_artman_config_filenames(
-            api_config_patterns, '*')
+            api_config_patterns, '*', [])
+    assert expected == actual
+
+
+def test_get_artman_config_filenames_wildcard_exclude():
+    api_config_patterns = ['test/cli/data/gapic/api/artman_${API_SHORT_NAME}.yaml',
+                           'test/cli/data/gapic/core/artman_${API_SHORT_NAME}.yaml']
+    exclude_apis = [
+        'test/cli/data/gapic/api/artman_longrunning.yaml',
+        'test/cli/data/gapic/core/artman_core.yaml',
+    ]
+    expected = [
+        'test/cli/data/gapic/api/artman_logging.yaml',
+        'test/cli/data/gapic/api/artman_pubsub.yaml',
+    ]
+    actual = batch_generation._get_artman_config_filenames(
+        api_config_patterns, '*', exclude_apis)
```

Comment:
Sorry I don't completely understand this. We just excluded logging and pubsub right? Why do we expect to see it?